numpy/core/src/multiarray/einsum.c.src

```diff
@@ -275,6 +301,8 @@ static void
     __m128 a, b;
 #elif EINSUM_USE_SSE2 && @float64@
     __m128d a, b;
+#elif EINSUM_USE_NEON_64 && @float64@
+    float64x2_t a, b;
```

Comment:
The new types are defined by the main extension header, e.g. [neon/neon.h](https://github.com/numpy/numpy/blob/master/numpy/core/src/common/simd/neon/neon.h).
however, in this case __m128d, and float64x2_t equivalent to npyv_f64.